### ISSUE TITLE
Investigate Ledger Support

### ISSUE THREAD

**Issue by avive** [karma-coin/karmachain#146]

**Comment by avive** [comment]
This is nice to have for tn3. how hard?

**Comment by HolyGrease** [comment]
> This is nice to have for tn3. how hard?

Do not know yet. Let you know as soon as possible

**Comment by HolyGrease** [comment]
As I can see Polkadot have Ledger support but with some issues like https://github.com/polkadot-js/extension/issues/1025#issuecomment-1066056797.

So the all supported interfaces are listed here: https://github.com/Zondax/ledger-polkadot

You can read on you own more here: https://wiki.polkadot.network/docs/ledger

I create an issue to ask about substrate compatibility here: https://github.com/Zondax/ledger-polkadot/issues/197.  But if it is I sure it works only for substrate based transaction, so in our case no `new_user` transaction no `appreciation` transaction.

**Comment by avive** [comment]
yes, I'm only interested in having support to simple coin transfer and mainly for validators.
